soc/arm/microchip_mec/mec1501/Kconfig.soc

```diff
@@ -70,4 +70,4 @@ config SOC_MEC1501_EXT_32K_PARALLEL_CRYSTAL
 config SOC_MEC1501_VTR3_1_8V
 	bool "VTR3 power rail is tied to 1.8V"
 	help
-	  Set this is if VTR3 power sourcejumper in the board is changed.
+	  Set this is if VTR3 power sourcejumper in the board is changed.
```

Comment:
This seems to have no change except the line return at the end.

Reply:
Done. Should be original version as I have not changes other than initial mistake.